crypto_kem/kyber1024/aarch64/api.h

```diff
@@ -1,3 +1,11 @@
+
+/*
+ * This file is licensed
+ * under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html) or
```

Comment:
Some of the files are MIT, some are Apache 2, is there a reason?

Reply:
We didn't change this file. The licensing is from where we got the file.